srcpkgs/cargo-bootstrap/template

```diff
@@ -28,31 +28,15 @@ esac
 case "$XBPS_TARGET_MACHINE" in
 	i686)
 		distfiles="${_bootstrap_url}/cargo-${version}-i686-unknown-linux-gnu.tar.xz"
-		checksum="e2e20a16f5db52cd6b773c94225b03880743544ff2f317dd857cf0dceac6ab57"
+		checksum="aeee38c0cd35b531d9f00028f720358256e9f3b80b176467c6afc033a16c92aa"
 		;;
 	x86_64)
 		distfiles="${_bootstrap_url}/cargo-${version}-x86_64-unknown-linux-gnu.tar.xz"
-		checksum="21434d83a30ad3fa4e4831487c5574a20b07dd57d213b26e1246290c57d4ec41"
+		checksum="b25d6f88b93cb75868ff4bc9ca0103facd4622825cf53df67546cea6cb60da0f"
 		;;
 	x86_64-musl)
 		distfiles="${_bootstrap_url}/cargo-${version}-x86_64-unknown-linux-musl.tar.xz"
-		checksum="01d06176fd894b9299ed4f5b78e87d78a9a25b9ba803d3f8c50b7a4ea21d8807"
-		;;
-	ppc64le)
```

Comment:
I'm not sure I see a reason to remove this? it won't work but if someone wants to put the effort into doing it again it can't hurt